### DISCUSSION TITLE
Fairy Ring Map Plugin

### DISCUSSION THREAD

**Discussion by CalvinKroese** [runelite/runelite#16228] [Ideas]
Looking for something to work on, is there a reason no one has done Fairy Rings menu similar to the [Nexus Menu Map](https://github.com/Antipixel/nexus-map)?

**Comment by Nightfirecat** [comment]
Can't think of one, should be doable

**Comment by ghost** [comment]
> Looking for something to work on, is there a reason no one has done Fairy Rings menu similar to the [Nexus Menu Map](https://github.com/Antipixel/nexus-map)?

poggers
